ProjectEureka/project-eureka-flutter
# PE-35 | Implemented New Question Pages

**What I did:**

- Created Screens to allow a user to create new questions posts.
- Create some components for things we might use often.

**How To Test :**

- Clone repo and run `flutter pub get`.
- Navigate from the side menu to get to the "Create a New Question Page".
- Select a category from the 4.
- Check that the selector on top will switch between "Text" "Photo" and "Video"
- Return to the "Text" tab and fill out a Question Title and Question Body.
-  Click Submit and watch you debug console, you should see the values you've entered: 
![image](https://user-images.githubusercontent.com/36316432/106521354-fa2a2180-6492-11eb-9de9-82b5f50dd451.png)
- You should then see a confirmation screen, make sure that you can see your title and body here again.
